packages/react-server/docs/why-react-server.md

```diff
@@ -31,8 +31,9 @@ the total time to full completion.
 In order to facilitate blazing perceived performance, `react-server` implements the following:
 
 * Parallelize backend queries
-* Use a client-side data cache
+* Data bundle transfer
```

Comment:
nitpick: this doesn't have parallel structure; the rest of the bullets are all actions (i.e. have a verb). How about "Bundle data requests" or "Bundle data for transfer"? 

And same for the section header below.